src/posts/what-is-wai.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What is WAI? The Web Accessibility Initiative
+description: "The Web Accessibility Initiative is W3C's effort to improve accessibility of the Web for people with disabilities"
+category: concepts
+author: Ilknur Eren
+date: 2022-07-18
+tags:
+  - concepts
+---
+
+## What is the World Wide Consortium?
+
+In order to know about the Web Accessibility Initiative, we must first talk about the World Wide Consortium, also known as W3C.
+
+Founded in 1994 and currently led by director Tim Berners-Lee and CEO Jeffrey Jaff, W3C is an international community where organizers develop international standards for the Web.
+
+As described by [W3’s website](https://www.w3.org/Consortium/), “W3C's mission is to lead the Web to its full potential.”
+
+Organizers from all around the world, involved in many different fields, join W3C to develop guidelines that ensure long-term growth for the Web. The Web Accessibility Initiative is one initiative that was launched by W3C.
+
+## What is the Web Accessibility Initiative?
+
+> The power of the Web is in its universality.
+> Access by everyone regardless of disability is an essential aspect.
+> Tim Berners-Lee, W3C Director and inventor of the World Wide Web
+
+W3C launched The Web Accessibility Initiative, (WAI), in 1997, with support from the White House. The Web Accessibility Initiative’s main goal is to make the internet accessible for people with disabilities.
+
+WAI brings people from disability organizations, governments and research labs from all around the world to develop guidelines and resources to make the Web accessible to people with disabilities that include auditory, cognitive, neurological, physical, speech and visual disabilities.
```

Comment:
```suggestion
WAI brings people from disability organizations, governments, and research labs from all around the world to develop guidelines and resources to make the Web accessible to people with disabilities. This includes: auditory, cognitive, neurological, physical, speech and visual disabilities.
```

This sentence was a little long, so I'm suggesting we split it into two smaller sentences.